AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -170,6 +171,18 @@ class CallingContext: NSObject {
         }
     }
 
+    func resumeVideo() {
+        if isVideoOnHold {
```

Comment:
We could add async completionHandler. Could take a reference from `endCall`
`resumeVideo(completionHandler: @escaping (Result<Void, Error>) -> Void)`